clang/include/clang/Serialization/ASTBitCodes.h

```diff
@@ -524,13 +524,7 @@ enum ASTRecordTypes {
   /// of source-location information.
   SOURCE_LOCATION_OFFSETS = 14,
 
-  /// Record code for the set of source location entries
-  /// that need to be preloaded by the AST reader.
-  ///
-  /// This set contains the source location entry for the
-  /// predefines buffer and for any file entries that need to be
-  /// preloaded.
-  SOURCE_LOCATION_PRELOADS = 15,
+  /// ID 15 used to be for source location entry preloads.
```

Comment:
Nit: this should be a normal `//` comment, because it doesn't attach to anything as a doc comment.